client/my-sites/hosting/php-version-card/index.js

```diff
@@ -0,0 +1,242 @@
+
+export const requestPhpVersion = siteId => {
+	const method = 'GET';
+
+	return requestHttpData(
+		requestId( siteId, method ),
+		http(
+			{
+				method,
+				path: `/sites/${ siteId }/hosting/php-version`,
+				apiNamespace: 'wpcom/v2',
+				body: {},
+			},
+			{}
+		),
+		{
+			fromApi: () => version => {
+				return [ [ requestId( siteId, method ), version ] ];
+			},
+			freshness: 0,
+		}
+	);
+};
+
+export const setPhpVersion = ( siteId, version ) => {
+	const method = 'POST';
+
+	return requestHttpData(
+		requestId( siteId, method ),
+		http(
+			{
+				method,
+				path: `/sites/${ siteId }/hosting/php-version`,
+				apiNamespace: 'wpcom/v2',
+				body: {
+					version: version,
+				},
+			},
+			{}
+		),
+		{
+			fromApi: () => success => {
+				if ( true === success ) {
+					requestPhpVersion( siteId );
+				}
+				return [ [ requestId( siteId, method ), success ] ];
+			},
+			freshness: 0,
+		}
+	);
+};
+
+const PhpVersionCard = ( {
+	translate,
+	siteId,
+	loading,
+	updating,
+	updateResult,
+	version,
+	showErrorNotice,
+	showSuccessNotice,
+} ) => {
+	const [ selectedPhpVersion, setSelectedPhpVersion ] = useState( null );
+
+	useEffect( () => {
+		requestPhpVersion( siteId );
+	}, [] );
+
+	useEffect( () => {
+		const updateNoticeId = 'hosting-php-version';
+
+		if ( updateResult === 'failure' ) {
+			showErrorNotice( translate( 'Failed to set PHP version.' ), {
+				id: updateNoticeId,
+			} );
+		}
+
+		if ( updateResult === 'success' ) {
+			showSuccessNotice(
+				translate( 'PHP version successfully set to %(version)s.', {
+					args: {
+						version: selectedPhpVersion,
+					},
+				} ),
+				{
+					id: updateNoticeId,
+					showDismiss: false,
+					duration: 5000,
+				}
+			);
+		}
+	}, [ updateResult ] );
+
+	const changePhpVersion = event => {
+		const newVersion = event.target.value;
+
+		setSelectedPhpVersion( newVersion );
+	};
+
+	const getPhpVersions = () => {
+		return [
+			{
+				label: translate( '7.2', {
+					comment: 'PHP Version for a version switcher',
+				} ),
+				value: '7.2',
+			},
+			{
+				label: translate( '7.3 (recommended)', {
+					comment: 'PHP Version for a version switcher',
+				} ),
+				value: '7.3',
+			},
+			{
+				label: translate( '7.4RC6', {
+					comment: 'PHP Version for a version switcher',
+				} ),
+				value: '7.4',
+			},
+		];
+	};
+
+	const getContent = () => {
+		if ( loading ) {
+			return;
+		}
+
+		const isButtonDisabled = ! selectedPhpVersion || selectedPhpVersion === version;
+		let buttonTooltip = undefined;
+		if ( isButtonDisabled ) {
+			buttonTooltip = translate( "You're already running PHP %(version)s.", {
+				args: {
+					version,
+				},
+			} );
+		}
+
+		return (
+			<div>
+				<p>{ translate( 'Manage PHP versions.' ) }</p>
+
+				<Accordion
+					title={ translate( 'How it affects your site?' ) }
+					className="php-version-card__accordion"
+				>
+					<p>{ translate( 'In all kinds of weird ways.' ) }</p>
```

Comment:
Is this the final copy? I was expecting something more informative for users 😅 We can always remove this accordion on this PR and follow up later once we have the final message.